include/i2c.h

```diff
@@ -58,6 +58,13 @@ extern "C" {
 /** Controller to act as Master. */
 #define I2C_MODE_MASTER			(1 << 4)
 
+/*
+ * The following #defines are used to configure the I2C slave device
+ */
+
+/** Slave device responds to 10-bit addressing. */
+#define I2C_SLAVE_FLAGS_ADDR_10_BITS		I2C_ADDR_10_BITS
```

Comment:
We don't use I2C_SLAVE_FLAGS_ADDR_10_BITS anywhere, so do we need it, or should the code in the slave files change to use it.